tests/nodeos_contrl_c_test.py

```diff
@@ -48,15 +50,17 @@
     # producer nodes will be mapped to 0 through totalProducerNodes-1, so the number totalProducerNodes will be the non-producing node
     specificExtraNodeosArgs[totalProducerNodes] = "--plugin eosio::producer_plugin --plugin eosio::chain_api_plugin --plugin eosio::http_plugin "
     "--plugin eosio::txn_test_gen_plugin --plugin eosio::producer_api_plugin "
+    traceNodeosArgs = " --plugin eosio::trace_api_plugin --trace-no-abis "
```

Comment:
Does Cluster require this now to work at all. If so, we should create an issue to just have Cluster automatically add this if not already added.

Reply:
See initial notes in new issue: https://github.com/eosnetworkfoundation/mandel/issues/285